text/0000-inline-asm.md

```diff
@@ -956,6 +956,12 @@ fn mul(a: u64, b: u64) -> u128 {
 }
 ```
 
+## Use AT&T syntax by default on x86
```

Comment:
This title says it uses AT&T syntax by default, but the paragraph says the RFC chooses to use Intel syntax by default.